core/services/aws/tests/commons/ddb/DynamoDbTableOperations.test.ts

```diff
@@ -0,0 +1,60 @@
+import DynamoDbTableOperations from '../../../commons/ddb/DynamoDbTableOperations'
+import { DynamoDBDocumentClient, PutCommand } from '@aws-sdk/lib-dynamodb'
+import { mockClient } from 'aws-sdk-client-mock'
+import { UserDTO } from '@commons/dto/UserDTO'
+import UserModel from '@application/technicalImpl/dbModels/UserModel'
+import DatabaseError from '@commons/libs/errors/DatabaseError'
+import { GENERIC_CODES } from '@commons/libs/constants/GenericCodes'
+import { mockUserWithStringId, expectedUser } from '@application/tests/mocks/mockUserData'
+
+const ddbMock = mockClient(DynamoDBDocumentClient)
+
+describe('DynamoDbTableOperations Tests', () => {
+  let dynamoDbOperations: DynamoDbTableOperations<UserDTO, any, UserModel>
```

Comment:
does this need to be let

Reply:
Yes, You are right this does not need to be let.